src/main.rs

```diff
@@ -55,7 +55,7 @@ fn display_error(errors: &[valgrind::xml::Error]) {
 fn main() {
     panic::replace_hook();
 
-    let number_of_arguments = || env::args_os().skip(0).count();
+    let number_of_arguments = || env::args_os().skip(1).count();
```

Comment:
This seems to change the behavior, doesn't it?

Reply:
It does, though I think it's what was intended. `args_os` has the first arg as the binary(path), and skip takes the number of elements to skip over.  I'm assuming the original intent was to get the number of arguments for the binary. `skip(0)` does nothing. `number_of_arguments` will always be at least 1.